clang/lib/CodeGen/CGCall.cpp

```diff
@@ -825,6 +831,47 @@ void computeSPIRKernelABIInfo(CodeGenModule &CGM, CGFunctionInfo &FI);
 }
 } // namespace clang
 
+ABIArgInfo CodeGenTypes::convertABIArgInfo(const llvm::abi::ABIArgInfo &abiInfo,
+                                           QualType type) {
```

Comment:
Clang is a bit inconsistent with this, but it generally also follows the "upper case variable names" style.